app/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
     package="hu.vmiklos.plees_tracker">
 
     <uses-permission android:name="android.permission.FOREGROUND_SERVICE" />
+    <uses-permission android:name="android.permission.READ_CALENDAR" />
+    <uses-permission android:name="android.permission.WRITE_CALENDAR" />
```

Comment:
As far as I see, you only import from the calendar, then drop the write permission?

Reply:
Sure - can do